testing/math.test

```diff
@@ -261,3 +261,107 @@ do_execsql_test divide-agg-int-agg-float {
 do_execsql_test divide-agg-float-agg-int {
   SELECT min(price) / min(id) from products
 } {1.0}
+
+
+do_execsql_test bitwise-and-int-null {
+  SELECT 1234 & NULL
+} {}
+
+do_execsql_test bitwise-and-int-int {
+  SELECT 1234 & 1234
+} {1234}
+
+do_execsql_test bitwise-and-int-float {
+  SELECT 660 & 261.8
+} {4}
+
+do_execsql_test bitwise-and-float-float {
+  SELECT 660.63 & 261.8
+} {4}
+
+do_execsql_test bitwise-and-float-int-rev {
+  SELECT 261.8 & 660
+} {4}
+
+do_execsql_test bitwise-and-float-int-rev {
+  SELECT SUM(id) from products 
```

Comment:
maybe i'm blind but this test doesn't have a bitwise operation?

Reply:
yup, used it to test the value for reference and forgot to delete it :)